src/Generator.Rewrite/RewriteOptions.cs

```diff
@@ -10,29 +10,47 @@ public class RewriteOptions
         /// <summary>
         /// Gets or sets the path to the target assembly that should be rewritten.
         /// </summary>
-        [Option('a', "assembly", Required = true,
-            HelpText = "The path to the target assembly that should be rewritten.")]
+        [Option
+        (
+            'a',
+            "assembly",
+            Required = true,
+            HelpText = "The path to the target assembly that should be rewritten."
+        )]
         public string TargetAssembly { get; set; }
 
         /// <summary>
-        /// Get or sets the path to the strong name key which should be used to sign or resign the assembly.
+        /// Gets or sets the path to the strong name key which should be used to sign or resign the assembly.
         /// </summary>
-        [Option('k', "signing-key",
-            HelpText = "The path to the strong name key which should be used to sign or resign the assembly.")]
+        [Option
+        (
```

Comment:
This is gonna be final style format for attrs? Look very strange begin with "[+\n+(" end with ")+]", imho

Reply:
Yes, the opening parens should probably share a line with the attribute name. It's not a perfect solution, but it does look a bit better.

Reply:
I wasn't sure which one to use here...
`[Option(` would kind of be against normal guidelines, but 
```
[Option
(
```

looks really weird.

So I guess this is an exception and I'll just use the first one instead